Src/PerformanceCollector/Shared/Implementation/QuickPulse/Process/PerfLib/NativeMethods.cs

```diff
@@ -0,0 +1,191 @@
+            // #endif
+            // so we can't use IntPtr here.
+
+            public int CounterNameTitlePtr = 0;
+            public int CounterHelpTitleIndex = 0;
+            public int CounterHelpTitlePtr = 0;
+            public int DefaultScale = 0;
+            public int DetailLevel = 0;
+            public int CounterType = 0;
+            public int CounterSize = 0;
+            public int CounterOffset = 0;
+        }
+
+        [StructLayout(LayoutKind.Sequential)]
+        internal class PERF_DATA_BLOCK
+        {
+            public int Signature1 = 0;
+            public int Signature2 = 0;
+            public int LittleEndian = 0;
+            public int Version = 0;
+            public int Revision = 0;
+            public int TotalByteLength = 0;
+            public int HeaderLength = 0;
+            public int NumObjectTypes = 0;
+            public int DefaultObject = 0;
+            public SYSTEMTIME SystemTime = null;
+            public int pad1 = 0;  // Need to pad the struct to get quadword alignment for the 'long' after SystemTime
+            public long PerfTime = 0;
+            public long PerfFreq = 0;
+            public long PerfTime100nSec = 0;
+            public int SystemNameLength = 0;
+            public int SystemNameOffset = 0;
+        }
+
+        [StructLayout(LayoutKind.Sequential)]
+        internal class PERF_INSTANCE_DEFINITION
+        {
+            public int ByteLength = 0;
+            public int ParentObjectTitleIndex = 0;
+            public int ParentObjectInstance = 0;
+            public int UniqueID = 0;
+            public int NameOffset = 0;
+            public int NameLength = 0;
+        }
+
+        [StructLayout(LayoutKind.Sequential)]
+        internal class PERF_OBJECT_TYPE
+        {
+            public int TotalByteLength = 0;
+            public int DefinitionLength = 0;
+            public int HeaderLength = 0;
+            public int ObjectNameTitleIndex = 0;
+            public int ObjectNameTitlePtr = 0;
+            public int ObjectHelpTitleIndex = 0;
+            public int ObjectHelpTitlePtr = 0;
+            public int DetailLevel = 0;
+            public int NumCounters = 0;
+            public int DefaultCounter = 0;
+            public int NumInstances = 0;
+            public int CodePage = 0;
+            public long PerfTime = 0;
+            public long PerfFreq = 0;
+        }
+      
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1812:AvoidUninstantiatedInternalClasses")]
+        [StructLayout(LayoutKind.Sequential)]
+        internal class SYSTEMTIME
+        {
+            public short wYear;
+            public short wMonth;
+            public short wDayOfWeek;
+            public short wDay;
+            public short wHour;
+            public short wMinute;
+            public short wSecond;
+            public short wMilliseconds;
+
+            public override string ToString()
```

Comment:
This is from CLR's NativeMethods.cs. I've cut the file down to only the structures/flags that we need, but I'd rather not make any changes unless necessary.